### PR TITLE
CashFlow progress tracking, remove unused clientToService stream

### PR DESCRIPTION
**Pull request by exFalso**

### CODE REVIEW

File: finance/src/main/kotlin/net/corda/flows/CashFlow.kt [hunk 1]
```diff
@@ -19,7 +20,16 @@ import java.util.*
...
  */
-class CashFlow(val command: CashCommand) : FlowLogic<CashFlowResult>() {
+class CashFlow(val command: CashCommand, override val progressTracker: ProgressTracker) : FlowLogic<CashFlowResult>() {
+    constructor(command: CashCommand) : this(command, tracker())
+
+    companion object {
+        object ISSUING : ProgressTracker.Step("Issuing cash")
+        object PAYING : ProgressTracker.Step("Paying cash")
+        object EXITING : ProgressTracker.Step("Exiting cash")
+
```
Comment (patrickkuo) [inline finance/src/main/kotlin/net/corda/flows/CashFlow.kt]: Do we need a error / failed step?

Comment (patrickkuo) [inline finance/src/main/kotlin/net/corda/flows/CashFlow.kt]: You are right (somehow I thought there are error steps somewhere in the code when I first comment on this).

Comment (patrickkuo) [inline finance/src/main/kotlin/net/corda/flows/CashFlow.kt]: LGTM, I will approve this after the conflicts are resolved.

Comment (exFalso) [inline finance/src/main/kotlin/net/corda/flows/CashFlow.kt]: Rebased

**Comment by mikehearn** [comment]
@patrickkuo I think the ball is in your court here.

**Pull request merged**